monai/bundle/utils.py

```diff
@@ -105,19 +105,43 @@
     "handlers_id": DEFAULT_HANDLERS_ID,
     "configs": {
         "tracking_uri": "$@output_dir + '/mlruns'",
+        "experiment_name": "default_experiment",
```

Comment:
```suggestion
        "experiment_name": "monai_experiment",
```
or default_monai_experiment (so that it's clear that it's from monai default API instead of mlflow's)